js/__internal/grids/grid_core/virtual_scrolling/m_virtual_scrolling.ts

```diff
@@ -776,18 +774,6 @@ const VirtualScrollingRowsViewExtender = (function () {
       dataController?.loadIfNeed?.();
     },
 
-    setColumnWidths(widths) {
-      const scrollable = this.getScrollable();
-      let $content;
-
-      this.callBase.apply(this, arguments);
-
-      if (this.option('scrolling.mode') === 'virtual') {
-        $content = scrollable ? $(scrollable.content()) : this.element();
-        this.callBase(widths, $content.children(`.${this.addWidgetPrefix(CONTENT_CLASS)}`).children(`:not(.${this.addWidgetPrefix(TABLE_CONTENT_CLASS)})`));
-      }
-    },
-
```

Comment:
We don't need this method because in if statement callBase is being called incorrectly. ColumnsView.setColumnWidths accepts options as parameter and not separate parameters as in 787 line.

Also the second parameter in the 787 line is the same tableElement which ColumnsView has